src/pages/resources/resources.ts

```diff
@@ -109,18 +110,49 @@ export const powerupPrice = derived(
     }
 )
 
-export const stakingPrice = derived(
+// price per kb
+export const netPowerupPrice = derived(
+    [msToRent, sampleUsage, statePowerUp, info],
+    ([$msToRent, $sampleUsage, $statePowerUp, $info]) => {
+        if ($msToRent && $sampleUsage && $statePowerUp) {
+            const price = $statePowerUp.net.price_per_kb($sampleUsage, $msToRent, $info)
+            return Asset.from(
+                $statePowerUp.net.price_per_kb($sampleUsage, $msToRent, $info),
+                '4,EOS'
+            )
+        }
+        return Asset.from(0, '4,EOS')
+    }
+)
+
+//price per ms
+export const cpuStakingPrice = derived(
     [activeBlockchain, msToRent, sampleUsage],
     ([$activeBlockchain, $msToRent, $sampleUsage]) => {
         if ($msToRent && $sampleUsage) {
-            const {account} = $sampleUsage
+            const { account } = $sampleUsage
             const cpu_weight = Number(account.total_resources.cpu_weight.units)
             const cpu_limit = Number(account.cpu_limit.max.value)
             let price = cpu_weight / cpu_limit
             if ($activeBlockchain.resourceSampleMilliseconds) {
```

Comment:
The resourceSampleMilliseconds of the telos chain is 1000, not sure if it has any impact